### PR TITLE
Added MVP Jenkins image

### PR DESCRIPTION
**Pull request by sabre1041**
Jenkins image

### CODE REVIEW

File: docker/jenkins/conf/configuration/jobs/bdd-coolstore-dsl/config.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<project>
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties/>
+  <scm class="hudson.plugins.git.GitSCM" plugin="git@2.4.4">
+    <configVersion>2</configVersion>
+    <userRemoteConfigs>
+      <hudson.plugins.git.UserRemoteConfig>
+        <url>https://github.com/sabre1041/ose-bdd-coolstore-demo.git</url>
```
Comment (jgoldsmith613) [inline docker/jenkins/conf/configuration/jobs/bdd-coolstore-dsl/config.xml]: I think this is still the old repo

File: docker/jenkins/conf/settings.xml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+      <id>mirrorId</id>
+      <mirrorOf>repositoryId</mirrorOf>
+      <name>Human Readable Name for this Mirror.</name>
+      <url>http://my.repository.com/repo/path</url>
+    </mirror>
+     -->
+   <mirror>
+      <id>nexus</id>
+      <mirrorOf>*</mirrorOf>
+      <url>http://nexus:8081/content/groups/public/</url>
```
Comment (jgoldsmith613) [inline docker/jenkins/conf/settings.xml:167]: Nothing wrong with this, but I find this cool that you can just put the service name as the url

File: docker/jenkins/scripts/kube-slave-common.sh [hunk 3]
```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+#
+# This file provides functions to automatically discover suitable image streams
+# that the Kubernetes plugin will use to create "slave" pods.
+# The image streams has to have label "role" set to "jenkins-slave".
+#
+# The Jenkins container also need permissions to access the OpenShift API to
```
Comment (jgoldsmith613) [inline docker/jenkins/scripts/kube-slave-common.sh:7]: Why do we have this file?  I thought you said we done have slaves

File: docker/jenkins/conf/configuration/jobs/bdd-coolstore-dsl/config.xml (hunk 1 above)
Comment (sabre1041) [inline docker/jenkins/conf/configuration/jobs/bdd-coolstore-dsl/config.xml]: good catch.

File: docker/jenkins/scripts/kube-slave-common.sh (hunk 3 above)
Comment (sabre1041) [inline docker/jenkins/scripts/kube-slave-common.sh:7]: Its here in case we decide to support it. i can remove it if we want

**Comment by sabre1041** [comment]
@jgoldsmith613 url updated and rebased

**Pull request merged**